source/minimessage.rst

```diff
@@ -152,10 +134,12 @@ Color (2, hex/rgb)
 
 A different, more flexible way (supports hex colors!) for colors looks like this
 
-Tag:
-   ``<color:_colorNameOrHex_>``  
-Arguments: 
-   * ``_colorNameOrHex_``, can be all the values from above, or hex colors (in 1.16)  
+Tag
+   ``<color:_colorNameOrHex_>``
+Aliases
+   ``colour``, ``c``
+Arguments
+   * ``_colorNameOrHex_``, can be all the values from above, or hex colors (in 1.16)
```

Comment:
`or hex colors` implying the above does **not** support hex which is not true anymore it seems